### PR TITLE
[ADD] Enable rounding for suppliers, configurable by a flag in the supplier

### PR DESCRIPTION
**Pull request by SimoRubi**
Hi, 
I've added the possibility to apply the rounding to supplier invoices too, based on a flag placed on the partner.
Please check, thanks

### CODE REVIEW

File: account_invoice_rounding/account.py [hunk 1]
```diff
@@ -174,13 +177,28 @@ def _compute_amount(self):
...
 
+    @api.multi
+    def onchange_partner_id(self, type, partner_id, date_invoice=False,
+                            payment_term=False, partner_bank_id=False,
+                            company_id=False):
+        result = super(AccountInvoice, self).onchange_partner_id(
+            type, partner_id, date_invoice=date_invoice,
+            payment_term=payment_term, partner_bank_id=partner_bank_id,
+            company_id=company_id)
+        partner = self.env['res.partner'].browse(partner_id or False)
```
Comment (tafaRU) [inline account_invoice_rounding/account.py]: what's the reason for `partner_id or False`? Better to check `partner_id` outside.

**Review by tafaRU — CHANGES_REQUESTED** [review]

File: account_invoice_rounding/account.py (hunk 1 above)
Comment (SimoRubi) [reply account_invoice_rounding/account.py]: thanks, fixed

File: account_invoice_rounding/partner.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class ModelName(models.Model):
```
Comment (tafaRU) [inline account_invoice_rounding/partner.py]: s/ModelName/ResPartner

Comment (SimoRubi) [reply account_invoice_rounding/partner.py]: thanks, fixed

**Comment by SimoRubi** [comment]
@tafaRU is this ok for you now?

**Comment by SimoRubi** [comment]
@tafaRU do you have any other correction to suggest about this module or is it ok?
thanks for the review

**Review by tafaRU — APPROVED** [review]

File: account_invoice_rounding/account.py [hunk 3]
```diff
@@ -174,12 +178,29 @@ def _compute_amount(self):
...
-    @api.one
+    @api.multi
     def _get_rounding_invoice_line_id(self):
-        lines = self.env['account.invoice.line'].search(
-            [('invoice_id', '=', self.id),
-             ('is_rounding', '=', True)])
-        self.global_round_line_id = lines
+        invoice_line_model = self.env['account.invoice.line']
+        for invoice in self:
+            lines = invoice_line_model.search(
```
Comment (moylop260) [inline account_invoice_rounding/account.py:184]: <s>What about use: `invoice.global_round_line_id = invoice.invoice_line_ids.filtered('is_rounding')`?</s> Forget it, It's version 8.0

**Review by moylop260 — APPROVED** [review]

**Comment by SimoRubi** [comment]
@eLBati could you please check? Thanks

**Comment by SimoRubi** [comment]
@eLBati is this ok for you?

**Review by eLBati — APPROVED** [review]

**Pull request merged**